extensions/dacpac/src/wizard/dataTierApplicationWizard.ts

```diff
@@ -324,11 +388,23 @@ export class DataTierApplicationWizard {
 			description: ''
 		};
 
-		let result = await service.generateDeployScript(this.model.filePath, this.model.database, ownerUri, azdata.TaskExecutionMode.script);
+		const result = await service.generateDeployScript(this.model.filePath, this.model.database, ownerUri, azdata.TaskExecutionMode.script);
 
 		if (!result || !result.success) {
 			vscode.window.showErrorMessage(loc.generateDeployErrorMessage(result?.errorMessage));
 		}
+
+		// Deploy Dacpac 'generate script' button clicked in DeployPlanPage, Reporting the event selection to the telemetry with fail/sucess status
+		TelemetryReporter.createActionEvent(TelemetryViews.DeployDacpac, 'GenerateDeployScriptOperation')
+			.withAdditionalProperties({
+				targetDatabaseStatus: 'Upgrade Existing Databse',
```

Comment:
why is this property needed? Generate script can only happen in this wizard when the target database exists, not for new databases

Reply:
Yeah, Not needed, just added as an extra information. Will remove it :)